### ISSUE TITLE
Custom Loss Objects and Dense should be built ahead of time.

### ISSUE THREAD

**Issue by Impure-King** [Impure-King/base-tensorwrap#6]
Currently, loss functions may only functions and not JIT compiled classes. This may improve customizability for users familiar with TensorFlow 2's inheritance, so it may be better to include them. Additionally, please build a sequential model ahead of time, in order to speed up performance and allow input shape predictions.

**Comment by Impure-King** [comment]
This method is currently error-prone and won't be implemented for a while.
